community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md

```diff
@@ -217,13 +222,13 @@ End-to-End Inference Time for AvatarChatbot Megaservice (asr -> llm -> tts -> an
 
 On SPR:  
 ~30 seconds for AudioQnA on SPR,  
-~40-200 seconds for AvatarAnimation on SPR
+~30-200 seconds for AvatarAnimation on SPR
```

Comment:
not sure about revealing performance without perf pdt approval... but no big concern from me.

Reply:
These are rough numbers